cmd/podman/parse/net.go

```diff
@@ -138,6 +138,8 @@ func parseEnvOrLabelFile(envOrLabel map[string]string, filename, configType stri
 	defer fh.Close()
 
 	scanner := bufio.NewScanner(fh)
+	buff := make([]byte, 256*(1<<10))
+	scanner.Buffer(buff, len(buff))
```

Comment:
This limit feels arbitrary. Can you document why this is chosen? I feel the correct way would be to stat the file and alloc the actual size?

Reply:
Yes, it is totally arbitrary.
TBH I updated this code only for testing sake. In reality long labels are not created via `--labels-file`, but by using image which has big labels.